collect_app/src/test/java/org/odk/collect/android/activities/FormDownloadListTest.java

```diff
@@ -0,0 +1,134 @@
+package org.odk.collect.android.activities;
+
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+import android.util.Log;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.odk.collect.android.logic.FormDetails;
+import org.odk.collect.android.preferences.PreferencesActivity;
+import org.robolectric.Robolectric;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+import java.util.HashMap;
+
+import io.ona.collect.android.BuildConfig;
+import io.ona.collect.android.R;
+import shadows.org.odk.collect.android.database.ShadowActivityLogger;
+import shadows.org.odk.collect.android.utilities.ShadowPRNGFixes;
+
+/**
+ * Tests for FormDownloadList that don't require instrumentation
+ *
+ * @author Jason Rogena - jrogena@ona.io
+ * @since 05/01/2017
+ */
+@RunWith(RobolectricTestRunner.class)
+@Config(constants = BuildConfig.class, sdk = 21, manifest = "src/main/AndroidManifest.xml",
+        shadows = {ShadowPRNGFixes.class, ShadowActivityLogger.class})
+public class FormDownloadListTest {
+
+    @Test
+    public void testGetOnaUser() {
+        String username1 = "dfsDFE12_fsd";
+        //test a normal url
+        String url1 = "https://odk.ona.io/"+username1+"/forms/129132/form.xml";
+        Assert.assertTrue(FormDownloadList.getOnaUser(url1).equals(username1));
+
+        //test http url
+        String url2 = "http://odk.ona.io/"+username1+"/forms/129132/form.xml";
+        Assert.assertTrue(FormDownloadList.getOnaUser(url2).equals(username1));
+
```

Comment:
let's add a test for "http://ona.io/"+username1+"/forms/129132/form.xml", this should be valid